src/screens/Settings/SettingsPage.js

```diff
@@ -77,3 +88,17 @@ export default class SettingsPage extends Component {
     );
   }
 }
+
+const mapStateToProps = state => ({
+  user: state.user,
+  errors: state.errors
```

Comment:
Do we end up using this `errors` state anywhere in this component?

Reply:
Not at the moment, but errors on logout should be written to it and referred to in the Toast notification.